polkadot-protocol/introduction/polkadot-networks.md

```diff
@@ -0,0 +1,82 @@
+
+This guide covers the tools and networks designed to support a wide variety of development and testing needs in the Polkadot ecosystem, from local development nodes to community-run TestNets.
+
+## Network Overview
+
+Polkadot's development process is a well-structured path designed to ensure that upgrades and new features undergo thorough testing before reaching the live production networks. This diagram illustrates the typical progression of a Polkadot development cycle, starting from local environments and eventually reaching the Polkadot MainNet. 
+
+``` mermaid
+---
+title: Polkadot Development Flow
+---
+flowchart LR
+    id1[Local] --> id2[Westend] --> id4[Kusama] --> id5[Polkadot]  
+    id1[Local] --> id3[Paseo] --> id4[Kusama] 
+```
+
+
+### Development Flow
+
+Application developers and end users of Polkadot ecosystem can test many features without risking or spending their tokens on production networks by using tools like [Chopsticks](#chopsticks) and TestNets. 
+
+A typical journey through the Polkadot development process might look like this:
+
+1. **Local Development Node** - for core Polkadot developers, the process begins in a local environment where you can work on upgrades and experiment with new features using a local development node. This environment allows for rapid iteration and testing in an isolated setup without the need for interacting with external networks
+
+2. **Westend** - once the upgrades are ready, they are deployed to [Westend](#westend), Polkadot's primary TestNet. Westend is designed to mimic the conditions of a live network without the risk of using real tokens. You can extensively test features here to ensure that everything works as expected before rolling them out to Kusama or Polkadot
+
+3. **Kusama** - after rigorous testing on Westend, the next step is Kusama. Kusama serves as Polkadot's experimental version and operates as a “canary network,” where features are deployed in a real-world environment with actual economic incentives. It's a high-fidelity testing ground for any potential network upgrade
+
+4. **Polkadot** - if an upgrade passes testing on Westend and Kusama, it is finally deployed to Polkadot, the MainNet. This workflow ensures that only thoroughly vetted changes make it to production
+
+Parachain developers use local TestNets powered by tools like [Zombienet](#zombienet) and then deploy the upgrades on their respective parachains to a TestNet.
+
+- **Paseo** - parachain developers often use Paseo, a community-run TestNet that mirrors Polkadot's runtime. Paseo provides parachain and dApp developers a space to test their applications with runtime environments that are closely aligned with Polkadot's without risking their projects on live networks
```

Comment:
Ah ok. Maybe we should just combine this with the Westend option and mention them both at the same time so it doesn't seem like there is preference to one or the other